.mocharc.js

```diff
@@ -1,7 +1,5 @@
-const os = require('os');
-
 module.exports = {
     colors: true,
     'enable-source-maps': true,
-    jobs: Math.min(2, os.cpus().length - 1),
+    require: [require.resolve('@stylable/e2e-test-kit/dist/browser-server.js')],
```

Comment:
Two options:
1. Move to package.json to not run in all tests. also add to the webpack-plugin 
2. look at process.cwd and enable when running from root or webpack-plugin

Reply:
1

Reply:
I choose 2 because it will give better experience when working on single package.